monai/visualize/class_activation_maps.py

```diff
@@ -17,24 +17,30 @@
 import torch.nn as nn
 import torch.nn.functional as F
 
+from monai.config import NdarrayTensor
 from monai.transforms import ScaleIntensity
 from monai.utils import ensure_tuple, get_torch_version_tuple
 from monai.visualize.visualizer import default_upsampler
 
 __all__ = ["CAM", "GradCAM", "GradCAMpp", "ModelWithHooks", "default_normalizer"]
 
 
-def default_normalizer(x) -> np.ndarray:
+def default_normalizer(x: NdarrayTensor) -> NdarrayTensor:
     """
     A linear intensity scaling by mapping the (min, max) to (1, 0).
+    The output data type and device will be consistent with input data.
 
```

Comment:
For the docstring, I don't think the data type should always be consistent here, input could be integers and output floats. Probably 'N.B.' is not a typo

Reply:
OK, that makes sense, I will update it.

Thanks.

Reply:
updated in the latest commit.
Thanks.